.gitignore

```diff
@@ -0,0 +1,3 @@
+mbed_cloud_dev_credentials.c
+update_default_resources.c
+upgradeCA.cert
```

Comment:
Do we need to ignore any of the other certificates that are deriived from mbed_cloud_dev_credentials.c?